checkov/terraform/checks/resource/kubernetes/LivenessProbe.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
+from checkov.common.models.consts import ANY_VALUE
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class LivenessProbe(BaseResourceValueCheck):
+
+    def __init__(self):
+        name = "Liveness Probe Should be Configured"
+        id = "CKV_K8S_8"
+        supported_resources = ["kubernetes_pod"]
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources,
+                         missing_block_result=CheckResult.FAILED)
+
+    def get_inspected_key(self) -> str:
+        return "spec/[0]/container/[0]/liveness_probe/[0]"
+
+    def scan_resource_conf(self, conf) -> CheckResult:
+        spec = conf.get('spec')[0]
+        if spec:
+            containers = spec.get("container")
+            for idx, container in enumerate(containers):
+                if type(container) != dict:
```

Comment:
```suggestion
                if not isinstance(container, dict):
```
most of the times it is better to use `isinstance` instead of `type` to check what kind of type a variable is, because it also includes subtypes.